### PR TITLE
Update statistical var #912

### PR DESCRIPTION
**Pull request by sp1103**
I modified the doc string for the var function in statistical. Looking at the function I did not see anything that needed reformatting. This is just a refactor of the doc string to add examples.

### CODE REVIEW

File: ivy/container/statistical.py [hunk 1]
```diff
@@ -88,6 +88,43 @@ def var(
...
+            inputs broadcast to.
+        Returns
+        -------
+        ret
+           if the variance was computed over the entire array, a zero-dimensional array
+            containing the variance; otherwise, a non-zero-dimensional array containing the
+            variances. The returned array must have the same data type as x.
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([0.1, 0.2, 0.9]), b=ivy.array([0.7, 0.1, 0.9]))
```
Comment (sherry30) [inline ivy/container/statistical.py:120]: put array `b` on a new line using `\` separator

File: ivy/container/statistical.py [hunk 2]
```diff
@@ -101,6 +138,68 @@ def var(
             out=out,
         )
 
+    def static_var(
```
Comment (sherry30) [inline ivy/container/statistical.py:141]: add `@staticmethod` above this

File: ivy/container/statistical.py [hunk 3]
```diff
@@ -101,6 +138,68 @@ def var(
...
+            inputs broadcast to.
+        Returns
+        -------
+        ret
+           if the variance was computed over the entire array, a zero-dimensional array
+            containing the variance; otherwise, a non-zero-dimensional array containing the
+            variances. The returned array must have the same data type as x.
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([0.1, 0.2, 0.9]), b=ivy.array([0.7, 0.1, 0.9]))
```
Comment (sherry30) [inline ivy/container/statistical.py:182]: put `b` on a new line using `\` separator

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,

I've requested a few changes on each specific line. The rest looks good, I'll merge as soon as you make these changes 😃 

Thanks

File: ivy/array/statistical.py [hunk 4]
```diff
@@ -44,6 +44,63 @@ def var(
...
+        containing the variance; otherwise, a non-zero-dimensional array containing the
+        variances. The returned array must have the same data type as x.
+    
+        Examples
+        --------
+
+        With :code:`ivy.Array` input:
+
+         >>> x = ivy.array([0.1, 0.2, 0.3, 0.3, 0.9, 0.10])
+         >>> y = ivy.var(x)
```
Comment (sherry30) [inline ivy/array/statistical.py:99]: you're supposed to add an instance method example here
`y = x.var()`

File: ivy/container/statistical.py [hunk 5]
```diff
@@ -88,6 +88,44 @@ def var(
...
+        Returns
+        -------
+        ret
+           if the variance was computed over the entire array, a zero-dimensional array
+            containing the variance; otherwise, a non-zero-dimensional array containing the
+            variances. The returned array must have the same data type as x.
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([0.1, 0.2, 0.9])\
+             b=ivy.array([0.7, 0.1, 0.9]))
```
Comment (sherry30) [inline ivy/container/statistical.py:121]: indent b exactly under a

File: ivy/container/statistical.py [hunk 6]
```diff
@@ -101,6 +139,70 @@ def var(
...
```
Comment (sherry30) [inline ivy/container/statistical.py:185]: indent b exactly under a

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,

just requested some minor changes,

Thanks

**Comment by sp1103** [comment]
Hi Sherry, 
I just noticed my pull request has not been merged. Is there anything else you need me to do.

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,

sorry this PR flew under my radar, you can request a review once you're done making changes, that will let me know its ready to be reviewed.

It seems like there are some lint errors in this file, can you fix those. Also test docstrings are failing, can you look into those and try to fix the errors.

If you have trouble fixing those then you can ask question, I'd be happy to help.
Thanks.

**Comment by sp1103** [comment]
Hey Sherry, 
I am trying to fix the lint errors but I keep getting E251 unexpected spaces around keyword / parameter equals. Also I was wondering if there was anything else I needed to do after I fix the lint error.

File: ivy/container/statistical.py [hunk 7]
```diff
@@ -101,6 +146,78 @@ def var(
...
+        ret
+           if the variance was computed over the entire array, 
+           a zero-dimensional array containing the variance; 
+           otherwise, a non-zero-dimensional array containing the 
+           variances. The returned array must have the same data 
+           type as x.
+       
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([0.1, 0.2, 0.9]),\
```
Comment (sherry30) [inline ivy/container/statistical.py:198]: write this as:
`x = ivy.Container(a=ivy.array([0.1, 0.2, 0.9]), \`
there should be space before and after the `\` separator.

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,
i pointed out a lint error, other than that, a line cannot be greater than 88 characters.

You also have to look at the docstring error and figure out why it is failing. Try to fix it or let me know if you are having trouble figuring it out.
Let me know if i can help you out with any other problem.

Thanks

**Comment by sp1103** [comment]
ivy_tests/test_ivy/test_functional/test_core/test_linalg.py:1280:9: E999 SyntaxError: keyword argument repeated

I keep getting this error for the linting other than that as far as I can tell everything should be in order.

File: ivy_tests/test_ivy/test_functional/test_core/test_linalg.py [hunk 8]
```diff
@@ -1277,4 +1277,6 @@ def test_diagonal(
         offset=offset,
         axis1=axes[0],
         axis2=axes[1],
-    )
\ No newline at end of file
+        x=np.random.uniform(size=(a, b)).astype(input_dtype[0]),
```
Comment (sherry30) [inline ivy_tests/test_ivy/test_functional/test_core/test_linalg.py:1280]: This line is extra

File: DockerfileGPU [hunk 9]
```diff
@@ -138,7 +138,7 @@ RUN pip3 install --no-cache-dir -r requirements.txt
 # Install local optional
 COPY optional.txt .
 RUN pip3 install --no-cache-dir -r optional.txt
-RUN pip3 install torch==1.10.0+cu111 -f https://download.pytorch.org/whl/torch_stable.html
+RUN pip3 install torch==1.11.0+cu111 -f https://download.pytorch.org/whl/torch_stable.html
```
Comment (sherry30) [inline DockerfileGPU:141]: Can you remove changes to this file in the commit

**Review by sherry30 — CHANGES_REQUESTED** [review]
I pointed out the extra line in your commit, can you remove that, I think you accidentally added it or maybe it got added while solving merge conflicts.

Also can you remove changes to the Docker file GPU as well. It shouldn't be in your commit.

Other than that, the docstring test for `var` is failing, the output on run seems to be a bit different from the output written in docs.

File: ivy_tests/test_ivy/test_functional/test_core/test_linalg.py [hunk 10]
```diff
@@ -1277,4 +1277,4 @@ def test_diagonal(
         offset=offset,
         axis1=axes[0],
         axis2=axes[1],
-    )
\ No newline at end of file
+    )
```
Comment (sherry30) [inline ivy_tests/test_ivy/test_functional/test_core/test_linalg.py:1280]: remove change from this file

File: ivy/functional/ivy/statistical.py [hunk 11]
```diff
@@ -439,7 +439,6 @@ def sum(
     >>> y = ivy.sum(x)
     >>> print(y)
     ivy.array(1.3)
-
```
Comment (sherry30) [inline ivy/functional/ivy/statistical.py:442]: remove change in this function

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,

Can you add 1 functional example using container as an input in `ivy/functional/ivy/statistical.py` and also 1 functional example using native array as input.
Some extra changes which i pointed out need to be removed which i think you accidentally pushed. 
Other than that it all looks good 😃

**Comment by sp1103** [comment]
Is it okay if we close this out

**Review by sherry30 — APPROVED** [review]
I've merged your PR, Thank you for the contribution 😄 
The PR will close automatically

**Pull request merged**